c increment and decrement operaters

```diff
@@ -0,0 +1,12 @@
+#include<stdio.h>
+int main ()
+{
+    int a=10,b =100
```

Comment:
Use Semicolon

Reply:
Anchal,just go through the changes.You have just left semicolon at 2 places.